lib/Braintree/OAuthGateway.php

```diff
@@ -70,7 +71,8 @@ public function _mapError($result)
         } else if ($error->code == Braintree_Error_Codes::OAUTH_INVALID_SCOPE) {
             $result->error = 'invalid_scope';
         }
-        $result->errorDescription = explode(': ', $error->message)[1];
+        $descriptions = explode(': ', $error->message);
+        $result->errorDescription = $descriptions[1];
```

Comment:
Should we ensure that `$descriptions` is an array and has two values to prevent notices?